packages/gatsby-plugin-fastify/src/plugins/server-routes.ts

```diff
@@ -60,16 +62,10 @@ export const handleServerRoutes: FastifyPluginAsync<{
           const pageData = (await renderPageData({ data: pageQueryData })) as unknown;
 
           if (page.mode === `SSR`) {
-            if (pageQueryData?.serverDataHeaders) {
-              reply.headers(pageQueryData.serverDataHeaders);
-            }
-
-            if (pageQueryData?.serverDataStatus) {
-              reply.code(pageQueryData.serverDataStatus);
-            }
+            if (pageQueryData?.serverDataStatus) reply.code(pageQueryData.serverDataStatus);
```

Comment:
Did a formatter change these or was that you? I'd prefer to leave the `{  }` on if statements.

Reply:
Cool, yeah I started to do this for a bit and realized it was always a pain. 🤣